libs/schema/src/index.ts

```diff
@@ -1,9 +1,29 @@
-import { Option as CliOption, OptionType } from '@angular/cli/models/interface';
+import { Schema } from '@nrwl/tao/src/shared/params';
+
+export enum OptionType {
+  Any = 'any',
+  Array = 'array',
+  Boolean = 'boolean',
+  Number = 'number',
+  String = 'string',
+}
+
+export type OptionPropertyDescription = Schema['properties'][number];
```

Comment:
This is what allows us to get the PropertyDescription type out of `Schema`. We can get the types out of a type by accessing the properties (ie. `Schema['properties']`). Since that type is an object, we can get the type out of the object by doing `[number]`. It's a bit weird, but that's advanced types for you 😅

Reply:
ok, yeah `number` was the part I'm not getting, but yeah it would be more straightforward if those types were exposed from Nx.